.gitlab-ci.yml

```diff
@@ -276,6 +276,7 @@ run_tests_ebpf:
   image: 486234852809.dkr.ecr.us-east-1.amazonaws.com/ci/datadog-agent-builders/ebpf:go1.10.1
   before_script:
     - cd $SRC_PATH
+    - pip3 install distro==1.4.0 # needed for some for our invoke tasks until the builders image contain it
```

Comment:
nit: why not use a full `pip3 install -r requirements.txt` here? So it's easy to pin the same version of `distro` as what's in `requirements.txt`.

Or alternatively `pip3 install distro -c requirements.txt` to only install `distro`

Reply:
Good idea thank you!